### ISSUE TITLE
Issue with EpicSpawners

### ISSUE THREAD

**Issue by VozikCZE** [BentoBoxWorld/Limits#155]
### Expected behavior

If I break stacked spawner, it should not decrease the count of spawner from limits.

### Observed/Actual behavior

If I break stacked spawner from EpicSpawners (actually I only remove one of them), it will decrease the amount of spawners from limits. There is problem with priority I think. It is better to see in video.

The biggest problem here is that player can have unlimited spawners.

### Steps/models to reproduce

[https://youtu.be/DxZ7Q9V0Ilw](https://youtu.be/DxZ7Q9V0Ilw)

### BentoBox version

[22:20:08 INFO]: Běžím na PURPUR 1.17.1.
[22:20:08 INFO]: Verze BentoBox: 1.18.1
[22:20:08 INFO]: Databáze: MYSQL
[22:20:08 INFO]: Načtené herní světy:
[22:20:08 INFO]: bskyblock_world (BSkyBlock): Svět, Nether, End
[22:20:08 INFO]: Načtené doplňky:
[22:20:08 INFO]: BSkyBlock 1.15.1 (ENABLED)
[22:20:08 INFO]: Chat 1.1.4 (ENABLED)
[22:20:08 INFO]: Greenhouses 1.6.0 (ENABLED)
[22:20:08 INFO]: Level 2.9.0 (ENABLED)
[22:20:08 INFO]: Limits 1.19.0 (ENABLED)
[22:20:08 INFO]: Warps 1.11.1 (ENABLED)

### Other

Source of EpicSpawners: https://github.com/songoda/EpicSpawners

**Comment by tastybento** [comment]
Yes. Right now Limits does not "know" about Epic Spawners. It would need to use the EpicSpawner's API to do that. That will take some work to do. I'll make this an Enhancement and hopefully someone can contribute some code if I don't manage to do it!

**Comment by tastybento** [comment]
**Assessment (July 2026):** Closing as not planned, same reasoning as #34/#92: per-plugin stacker integrations (EpicSpawners here) tie Limits to third-party APIs and their release cycles for a single plugin pairing, which doesn't fit the addon's maintenance goals.

Worth noting the concrete exploit in the video (break one of a stack, count decrements below reality) is inherent to any stacker without integration — the practical mitigation is a periodic admin recount, or a small bridge plugin using EpicSpawners' API to apply island offsets.
